package.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "@exabyte-io/esse.js",
-    "version": "2021.03.15.2",
+    "version": "2021.03.15.3",
     "description": "Exabyte Source of Schemas and Examples",
     "scripts": {
         "prepublishOnly": "rm -rf lib; npm run transpile",
-        "transpile": "mkdir -p lib; babel-compile -p es2015 src/js:lib/js",
-        "test": "mocha $NODE_DEBUG_OPTION  --bail --require babel-register src/js/esse/tests/*.js"
+        "transpile": "mkdir -p lib; babel --out-dir lib/js src/js;",
```

Comment:
Just to clarify - do we not need to pass es2015 as babel preset explicitly anymore?

Reply:
Looks like the the yearly presets were deprecated in babel 6 in favor of `preset-env`
https://babeljs.io/docs/en/babel-preset-es2015/

You can see that replacement above in .babelrc